packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1306,21 +1387,21 @@ class Container extends StatelessWidget {
   /// If null, the container will expand to fill all available space in its parent.
   final Widget child;
```

Comment:
should we have a `static final Container empty = new Container(width: 0.0, height: 0.0)` ?